### ISSUE TITLE
Feature request: easy to implement combine --unique --count to compute number unique of items 

### ISSUE THREAD

**Issue by Zer0-Tolerance** [lizmat/App-Rak#39]
Hi Liz,
it seems that in the latest version the `--unique --count`  is not working as one would expect:
`rak "/./" file.txt --unique --count-only` => should give the number of unique pattern but it only gives the count of non unique. 

Or am I missing something ?

**Comment by lizmat** [comment]
Hmmm... I seem to recall that `--count-only` overrides `--unique`, but maybe it shouldn't.

Lemme mull on that...  :-)

**Comment by lizmat** [comment]
```
$ raku -e 'say (1 / 998.001).Rat.Str(:300digits)'        
0.0010020030040050060070080090100110120130140150160170180190200210220230240250260270280290300310320330340350360370380390400410420430440450460470480490500510520530540550560570580590600610620630640650660670680690700710720730740750760770780790800810820830840850860870880890900910920930940950960970980991
```
Note that the final "100" is shown as "1" because of `0`'s being removed from the tail.  Perhaps a specific `:digits` specification should overrule that.
```
$ raku -e 'say (1 / 998.001).Rat.Str(:303digits)'        
0.001002003004005006007008009010011012013014015016017018019020021022023024025026027028029030031032033034035036037038039040041042043044045046047048049050051052053054055056057058059060061062063064065066067068069070071072073074075076077078079080081082083084085086087088089090091092093094095096097098099100101
```
shows that the correct digits *are* there.

**Comment by lizmat** [comment]
This will requires some internal re-design.  Mulling again.

**Comment by Zer0-Tolerance** [comment]
if one wants to get a count of unique values then you need to pipe it to an external command as sort -u if you use `--count-only` or `wc -l` if you use `--unique`  ... it would really be better to have it in rak.

**Comment by lizmat** [comment]
Still mulling, didn't make it to the [0.2.24](https://raku.land/zef:lizmat/App::Rak?v=0.2.24) release  :-(

**Comment by Zer0-Tolerance** [comment]
ok maybe the next one ?

**Comment by Zer0-Tolerance** [comment]
up

**Comment by lizmat** [comment]
Fixed in release [0.3.7](https://raku.land/zef:lizmat/App::Rak?v=0.3.7)
